backend/endpoints/rooms.py

```diff
@@ -61,3 +63,69 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+
+@router.put(
+    "/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    status_code=status.HTTP_200_OK,
+    responses={
+        400: {"detail": "the max number for a Group_DM is 9"},
+        401: {"detail": "member not an admin"},
+        403: {"detail": "DM room or not found"},
+    },
+)
+async def add_to_room(data: AddToRoom, org_id: str, room_id: str, member_id: str):
+    """Adds a new member(s) to a room
+    Args:
+        data: a pydantic schema that defines the request params
+        org_id (str): A unique identifier of an organisation
+        room_id: A unique identifier of the room to be updated
+
+    Returns:
+        HTTP_200_OK: member added
+    Raises:
+        HTTP_400_BAD_REQUEST: the max number for a Group_DM is 9
+        HTTP_401_UNAUTHORIZED: member not an admin
+        HTTP_403_FORBIDDEN: DM room or not found
+    """
+
+    new_member = data.dict().get("new_member")
+    room = await get_room(org_id=org_id, room_id=room_id)
+
+    if room is not None and room["room_type"] != RoomType.DM:
```

Comment:
What do you think if the nestings were reduced and nested ifs were avoided. This isn't consequential though, but might make the code easily readable.

Something like
`
if room is None and room['room_type'] == RoomType.DM:
      raise HTTPException
if member['role'] != Role.ADMIN:
       raise HTTPException

`

Reply:
hmmm, I saw what you did there. I think I agree too. I will see how I can make that work.